settings.gradle

```diff
@@ -1,3 +1,4 @@
 include ':hugo-annotations'
 include ':hugo-plugin'
 include ':hugo-runtime'
+include ':hugo-example'
```

Comment:
This should not be here. The project cannot run in the same Gradle invocation as it depends on the plugin.